mmv1/third_party/terraform/tests/resource_network_services_http_route_test.go.erb

```diff
@@ -0,0 +1,83 @@
+<% autogen_exception -%>
+package google
+<% unless version == 'ga' -%>
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccNetworkServicesHttpRoute_update(t *testing.T) {
+	t.Parallel()
+
+	httpRouteName := fmt.Sprintf("tf-test-http-route-%s", RandString(t, 10))
+
+	VcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
```

Comment:
testAccPreCheck has to be changed to AccTestPreCheck

Reply:
fixed.